_article-types/howto.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: article-type
+name: howto
```

Comment:
Yeah just tested this and it doesn't work. From your link:

> Collections have the following template variables **available for permalinks**

It already seemed weird to me and I looked at it yesterday and found this: http://stackoverflow.com/questions/13243469/how-can-a-jekyll-page-access-its-filename